src/main/resources/com/google/api/codegen/nodejs/sample.snip

```diff
@@ -0,0 +1,108 @@
+@snippet generate(sampleClass)
+  // BEFORE RUNNING:
+  // ---------------
+  // 1. If not already done, enable the {@sampleClass.apiTitle}
+  //    and check the quota for your project at
+  //    https://console.developers.google.com/apis/api/{@sampleClass.apiName}
+  // 2. Install the Node.js client library by running
+  //    `npm install googleapis --save`
+  // 3. This sample uses Application Default Credentials for authentication.
+  //    If not already done, install the gcloud CLI from
+  //    https://cloud.google.com/sdk and run
+  //    `gcloud beta auth application-default login`
+
+  var google = require('googleapis');
+  var GoogleAuth = require('google-auth-library');
+
+  var {@sampleClass.body.serviceVarName} = google.{@sampleClass.body.serviceTypeName}('{@sampleClass.apiVersion}');
+
+  {@mainFunc(sampleClass.body)}
+
+  {@authFunc()}
+@end
+
+@private mainFunc(sampleBody)
+  authorize(function(authClient) {
+    var {@sampleBody.requestVarName} = {
+      @join field : sampleBody.fields if sampleBody.fields
+        @if field.description
+          @join line : util.getDocLines(field.description, 100)
+            // {@line}
+          @end
+
+
+        @end
+        {@field.name}: {@field.defaultValue}, // TODO: Update placeholder value(s).
+
+      @end
+      @if sampleBody.hasRequestBody
+      resource: {
+        // TODO: Add desired properties of the request body.
+      },
+
+      @end
+      auth: authClient
+    };
+
+    @if sampleBody.isPageStreaming
+      var handlePage = function(err, response) {
+        if (err) {
+          console.log(err);
+          return;
+        }
+
+        var page = response['{@sampleBody.resourceFieldName}'];
```

Comment:
Please snippetize variable names like `response` as in discovery_fragment.snip, to make them easy to change.